opengrok-web/pom.xml

```diff
@@ -136,18 +136,28 @@
                     <path>/${project.build.finalName}</path>
                 </configuration>
             </plugin>
-
-            <plugin>
-                <groupId>org.apache.maven.plugins</groupId>
-                <artifactId>maven-surefire-plugin</artifactId>
-                <configuration>
-                    <argLine>${surefireArgLine}</argLine>
-                </configuration>
-            </plugin>
```

Comment:
again, this can stay here and you can update just the argline in the profile

Reply:
what do you mean exactly ?